driver/driver_options_test.go

```diff
@@ -94,16 +97,50 @@ func TestDriverOptions_writeTimeout(t *testing.T) {
 	srv := CreateMockServer(t)
 	defer srv.Stop()
 
-	conn, err := sql.Open("mysql", "root@127.0.0.1:3307/test?writeTimeout=10")
```

Comment:
I see that in the old code, `10` is an illegal value and should return error. So line 102 catched it. Now what's the purpose to change it to `1ns`?

Reply:
> I see that in the old code, `10` is an illegal value and should return error. So line 102 catched it. Now what's the purpose to change it to `1ns`?

I'm going to add some comments into the test to explain. But you're right in that i unintentionally altered what was being tested so I'll fix that as well.